Documentation/UPI/vSphere_terraform/README.md

```diff
@@ -0,0 +1,201 @@
+# vSphere UPI
+
+This guide explains how to provision Fedora CoreOS on vSphere and install OKD on it. It provides terraform files to create and destroy a cluster. This terraform files are based on the [CI-platform files](https://github.com/openshift/installer/tree/fcos/upi/vsphere), but are much simpler and doesn't include communication with other infrastructure components except vSphere.
+
+## Pre-Requisites
+
+* terraform > v0.12
+
+## Prepare environment
+
+Checkout the official [Openshift 4.3 Documentation](https://docs.openshift.com/container-platform/4.3/installing/installing_vsphere/installing-vsphere.html#installation-requirements-user-infra_installing-vsphere). Add DNS and DHCP records for your virtual machines, that will be provisioned in next steps.
+
+Install a loadbalancer in front of your API servers. Choose a high availability setup, if you need it. The loadbalancer is only for API traffic and Kubernetes can handle an outage of the API. During an outage the cluster is not able to change something.
+
+Find, download and upload an image of FCOS for [VMware vSphere](https://getfedora.org/en/coreos/download/).
+
+### Example Environment
+
+This example environment includes three master and worker nodes. Etcd is running on master nodes.
+
+#### A-records with mac address
+
+| FQDN | A-record | MAC address |
+|------|----------|-------------|
+| bootstrap.okd.example.com | 10.20.15.3 | 00:1c:14:00:00:03 |
+| lb.okd.example.com | 10.20.15.2 | 00:1c:14:00:00:02 |
+| api.okd.example.com | 10.20.15.2 | |
+| api-int.okd.example.com | 10.20.15.2 | |
+| master1.okd.example.com | 10.20.15.11 | 00:1c:14:00:00:11 |
+| master2.okd.example.com | 10.20.15.12 | 00:1c:14:00:00:12 |
+| master3.okd.example.com | 10.20.15.13 | 00:1c:14:00:00:13 |
+| etcd-0.okd.example.com | 10.20.15.11 | |
+| etcd-1.okd.example.com | 10.20.15.12 | |
+| etcd-2.okd.example.com | 10.20.15.13 | |
+| worker1.okd.example.com | 10.20.15.41 | 00:1c:14:00:00:41 |
+| worker2.okd.example.com | 10.20.15.42 | 00:1c:14:00:00:42 |
+| worker3.okd.example.com | 10.20.15.43 | 00:1c:14:00:00:43 |
+
+#### SRV-records
+
+| domain | TTL | priority | weight | port | target |
+|--------|-----|----------|--------|------|--------|
+| _etcd-server-ssl._tcp.okd.example.com | 86400 | 0 | 10 | 2380 (etcd) | 10.20.15.11, 10.20.15.12, 10.20.15.13
+
+#### wildcard DNS record
+
+| domain | A-record | comment |
+|--------|----------|---------|
+| *.apps.okd.example.com | 10.20.15.41, 10.20.15.42, 10.20.15.43 | For high availability, we choose the first three worker nodes, which should run the router component |
+
+#### Loadbalancer - HAProxy configuration
```

Comment:
Maybe include href to https://github.com/openshift/okd/blob/master/Documentation/UPI/Requirements/DNS_Bind.md & https://github.com/openshift/okd/blob/master/Documentation/UPI/Requirements/LB_HAProxy.md for anyone who might need more detail?

Reply:
I doesn't like the [HAProxy Configuration](https://github.com/openshift/okd/blob/master/Documentation/UPI/Requirements/LB_HAProxy.md) as mentioned above. In my infrastructure I have a DNS-GUI, but I think it's worth to add a link to a bind/named configuration.

Reply:
Sure, no worries then.